### PR TITLE
project weather app

### PR DESCRIPTION
**Pull request by idsab**
## Netlify link
Add your Netlify link here.
https://vermillion-valkyrie-43120b.netlify.app/

### CODE REVIEW

File: code/script.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
+//store API+KEY in variable
+const weatherUrl = `https://api.openweathermap.org/data/2.5/weather?q=Stockholm,Sweden&units=metric&APPID=764dd5634dc2ea4c9de71e7b62436c65
+
+`;
+const forecastUrl = `https://api.openweathermap.org/data/2.5/forecast?q=Stockholm,Sweden&units=metric&APPID=764dd5634dc2ea4c9de71e7b62436c65
```
Comment (Xikitita) [inline code/script.js:5]: These two URLs look identical. Could one of them be reduced?

File: code/script.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+      const description = json.weather[0].description;
+      const temperature = json.main.temp.toFixed(1); //temp value and round of to one decimal
+      const sunrise = new Date(json.sys.sunrise * 1000).toLocaleTimeString([], {
+        hour: "2-digit",
+        minute: "2-digit",
+      });
+      const sunset = new Date(json.sys.sunset * 1000).toLocaleTimeString([], {
+        hour: "2-digit",
+        minute: "2-digit",
+      });
```
Comment (Xikitita) [inline code/script.js:35]: Really nice and clean scope! It's readable and easy to understand. Clean fetching of the sunrise/sunset times.

File: code/style.css [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+body {
+  font-family: Montserrat;
+  display: flex;
+  justify-content: center;
+  background-color: lightpink;
+}
+
+#container {
```
Comment (Xikitita) [inline code/style.css:13]: The responsiveness up to around 480px looks great! Could the container it be stretched in height to keep the nice looking format and avoid the cube-shape that occurs from 500px? Just a design preference from my side!

**Review by Xikitita — COMMENTED** [review]
Over all amazing job! Clean and readable code, nice designs and you managed one of the stretch goals even though I'm aware you were super short on time. Impressive!

File: code/style.css
Comment (HIPPIEKICK) [inline code/style.css:1]: Consider using the whole width on small devices as it otherwise gets cramped:
Skärmavbild 2024-03-12 kl  15 02 45

**Review by HIPPIEKICK — APPROVED** [review]

**Pull request closed without merging**